### PR TITLE
HV:debug:profiling Fixed inappropriate condition check

### PR DESCRIPTION
**Pull request by mchinth**
Fixed the condition to be able to collect MSR sample data
Also populating addition information to help identifity appropriate data

Tracked-On: #1693
Signed-off-by: Manisha Chinthapally <manisha.chinthapally@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
1695<br>No new violations to the coding guideline detected.

File: hypervisor/include/debug/profiling_internal.h [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #define SEP_BUF_ENTRY_SIZE 		32U
 #define SOCWATCH_MSR_OP			100U
 
+#define MAGIC_NUMBER			99999988U
```
Comment (dongyaozu) [inline hypervisor/include/debug/profiling_internal.h]: usually people uses hex # for magic. What is the purpose of decimal here?

Comment (dongyaozu) [reply hypervisor/include/debug/profiling_internal.h]: anyway, hex is better for debug

**Comment by HuiHuangShi** [comment]
1695<br>No new violations to the coding guideline detected.

**Comment by HuiHuangShi** [comment]
1695<br>No new violations to the coding guideline detected.

File: hypervisor/include/debug/profiling_internal.h (hunk 1 above)
Comment (mchinth) [reply hypervisor/include/debug/profiling_internal.h]: Updated to HEX

**Pull request merged**